packages/bpk-tokens/src/ios/base/aliases.json

```diff
@@ -20,7 +20,6 @@
     "FONT_WEIGHT_NORMAL": "400",
     "FONT_WEIGHT_SEMIBOLD": "600",
     "FONT_WEIGHT_BOLD": "700",
-    "FONT_WEIGHT_HEAVY": "800",
```

Comment:
This was unused - aliases are not included in output (apart from raw but these token formats are not "advertised") so non breaking change.